app/models/task.py

```diff
@@ -1,5 +1,39 @@
+import datetime
 from app import db
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement = True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable = True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'))
+    goal = db.relationship("Goal", back_populates = "tasks")
+
+    @classmethod
+    def from_json(cls, req_body):
+        return cls(
+            title = req_body["title"],
+            description = req_body["description"]
+            ) 
+
+    def to_dict(self):
+        ret = {
+                "task": {
+                "id": self.task_id,
+                "title": self.title,
+                "description": self.description,
+                "is_complete": False
```

Comment:
How could we make this more flexible, so that it will change to `True` or `False`depending on the Task's `completed_at` attribute. Once we do that, we can use this for all of our routes, not just when we first make a brand new Task.

Maybe some kind of conditional statement would help with that.